run_dir/design/flowcells.html

```diff
@@ -11,7 +11,15 @@
 
 <div class="container-fluid" id="flowcell_list">
   <div id="fc_table_div">
-      <h1><span id="page_title">Flowcells</span> </h1>
+      <h1><span id="page_title">Flowcells</span>
+          {% if all %}
```

Comment:
The position of the button jumps on the page when switching between the options. Should we do something about it? Maybe @ewels would have a better idea.